ledger/accountdb.go

```diff
@@ -938,3 +897,66 @@ func (mc *merkleCommitter) LoadPage(page uint64) (content []byte, err error) {
 func (mc *merkleCommitter) GetNodesCountPerPage() (pageSize int64) {
 	return merkleCommitterNodesPerPage
 }
+
+type encodedAccountsIterator struct {
+	rows            *sql.Rows
+	orederByAddress bool
+}
+
+// encodedAccountsRange returns an array containing the account data, in the same way it appear in the database
+// starting at entry startAccountIndex, and up to accountCount accounts long.
+func (iterator *encodedAccountsIterator) Iterate(ctx context.Context, tx *sql.Tx, accountCount int) (bals []encodedBalanceRecord, err error) {
```

Comment:
Please update the comment, there is no `encodedAccountsRange` anymore.

Strictly speaking, `Iterator` usually has `Next/Advance` method, or accepts a callback. Here we return batch of all available elements. Suggestion: call it `encodedAccountsBatchIter` and rename `Iterate` to `Next` or `NextBatch`